eng/pipelines/templates/steps/test.yml

```diff
@@ -50,7 +50,7 @@ steps:
 
   - ${{ if eq(parameters.TestProxy, true) }}:
     - pwsh: |
-        cat $(Build.SourcesDirectory)/test-proxy.log
+        copy $(Build.SourcesDirectory)/test-proxy.log $(Build.ArtifactStagingDirectory)/test-proxy.$(jobSuffix).log
```

Comment:
Do you need a suffix? I assume as long artifact name is unique that is be all that is needed.

Reply:
Right this can be the same.